Zend/tests/restrict_globals/invalid_assign_list_ref.phpt

```diff
@@ -0,0 +1,10 @@
+--TEST--
+Cannot list-assign to $GLOBALS (by-ref)
+--FILE--
+<?php
+
+list(&$GLOBALS) = [1];
+
+?>
+--EXPECTF--
+Fatal error: Cannot assign reference to non referencable value in %s on line %d
```

Comment:
Out of curiosity: it is `referencable` or `referenceable` (or both)?

As you can see here, https://github.com/php/php-src/blob/5bd7305cf0847ad6aa13ced87f83091d4753ee64/Zend/zend_vm_def.h#L2334 is using the word `referenceable`.

There are a few other places that use the `referencable` word. 

If `referenceable` is the proper option, I can submit a PR that changes those files.

Reply:
It seems that both spellings are accepted, but google prefers referenceable.